src/pages/trade-options/option-tabs/option-tabs.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../variables";
```

Comment:
The overall width of the tabs should be the same as the table with the options underneath. It looks a little weird otherwise.

![image](https://user-images.githubusercontent.com/14966470/93194661-4c3c0600-f783-11ea-831f-0772b687636a.png)